lib/storage/src/content_manager/conversions.rs

```diff
@@ -30,9 +31,11 @@ impl TryFrom<api::grpc::qdrant::CreateCollection> for CollectionMetaOperations {
             CreateCollection {
                 vectors: match value.vectors_config.and_then(|config| config.config) {
                     Some(vector_config) => vector_config.try_into()?,
-                    None => return Err(Status::invalid_argument("vectors config is required")),
```

Comment:
where is the validation taking place assuring that `vectors` OR `sparse_vectors` are defined for gRPC?

Reply:
As TODO comment, it's an untrivial moment. We have to validate 2 separate fields with 10 another ones with regular validation - manual implementation would be large =) Moreover, after discussion with @generall, collection without any vectors are valid. added one more todo comment here

Reply:
sorry I am a bit a confused now.
if collection without any vectors are valid, do still need a TODO?